tests/yam/validate/validate_ha.pm

```diff
@@ -0,0 +1,22 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate that the product installed by agama via /etc/os-release
```

Comment:
this only validate the repo not the /etc/os-release, I remember we already have some module to validate repos from other test suites.

Reply:
additionally to validate that is minimal or not I think we have some other module to check patterns.

Reply:
Maybe this tests/console/validate_repos.pm can be used instead.

Reply:
This tests/console/validate_packages_and_patterns.pm is for validate packages and patterns, need check whether can be used for our test.

Reply:
Switched to `validate_repos` now as a validation module. This introduced a new test data file.